syntax/internal/stdlib/stdlib.go

```diff
@@ -86,7 +94,8 @@ var str = map[string]interface{}{
 }
 
 var array = map[string]interface{}{
-	"concat": concat,
+	"concat":       concat,
+	"combine_maps": mapCombine,
```

Comment:
```suggestion
	"combine_maps": combineMaps,
```
Let's rename the code to match to keep cognitive load under control.